### PR TITLE
run tests on Linux using powershell

### PR DESCRIPTION
**Pull request by friism**
ahhh :-)

### CODE REVIEW

**Comment by friism** [comment]
@MichaelSimons this will require CI update for debian. Let me know how you want to do it - I can check the change in and leave the `.sh` files so that we can divvy it up in stages, or you can validate manually.

Let me know.

File: test/Dockerfile.linux.testrunner [hunk 1]
```diff
@@ -15,3 +15,6 @@ RUN apt-get update \
 RUN curl -o powershell.deb -ssL https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.15/powershell_6.0.0-alpha.15-1ubuntu1.16.04.1_amd64.deb \
 	&& dpkg -i powershell.deb \
 	&& rm -f powershell.deb
+
+WORKDIR /test
+ADD . .
```
Comment (MichaelSimons) [inline test/Dockerfile.linux.testrunner]: Why use ADD?  I thought [Dockerfile best practices ](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy) prefer COPY?

File: test/Dockerfile.linux.testrunner [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Use this Dockerfile to create a Linux test-runner image
-#     docker build -t testrunner -f .\test\Dockerfile.runner .\test\.
-#     docker run -v /var/run/docker.sock:/var/run/docker.sock -v ${PWD}:/run --workdir /run testrunner powershell -File build-and-test.ps1 -Platform linux -UseImageCache
+# Use this Dockerfile to create a Linux testrunner image
+#     docker build -t testrunner -f .\test\Dockerfile.linux.testrunner .
```
Comment (MichaelSimons) [inline test/Dockerfile.linux.testrunner]: Using a '/' seems more useful since this exists for linux scenarios.

**Comment by friism** [comment]
@MichaelSimons sent replacement: https://github.com/dotnet/dotnet-docker/pull/194

**Pull request closed without merging**